pylabrobot/resources/carrier.py

```diff
@@ -241,7 +240,7 @@ def get_plate_sinking_depth(plate: Plate):
       resource.register_did_assign_resource_callback(self._update_resource_stack_location)
       self.register_did_unassign_resource_callback(self._deregister_resource_stack_callback)
 
-    return get_child_location(resource) - Coordinate(z=z_sinking_depth)
```

Comment:
_get_child_location would mean the location is explicitly specified, which should mean that the mixin doesn't apply (that's only for default values). so I think we should keep get_child_location here

Reply:
seems like the opposite case in line 209?
```
    location = location or self._get_child_location(resource)
```